### PR TITLE
Remove warnings of memory leak and 'read property offsetWidth of null…

### PR DESCRIPTION
**Pull request by redzumi**
…' message

- [x] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [x] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -283,12 +283,14 @@ export default class Carousel extends Component {
...
      const arrowRightWidth = this.arrowRightNode && this.arrowRightNode.offsetWidth;
-     const width = this.node.offsetWidth - (arrowLeftWidth || 0) - (arrowRightWidth || 0);
-
-     this.setState(() => ({
-       carouselWidth: width,
-       windowWidth: window.innerWidth,
-     }));
+     const width = this.node && this.node.offsetWidth - (arrowLeftWidth || 0) - (arrowRightWidth || 0);
+
+     if (this.node) {
```
Comment (piotr-s-brainhub) [inline src/components/Carousel.js]: it would be simpler to write
```
if (!this.node) {
  return;
}
```
at the begin of this function

Comment (redzumi) [reply src/components/Carousel.js]: yep, you right, fixed

**Comment by piotr-s-brainhub** [comment]
Thanks @redzumi for this PR.
@Lukasz-pluszczewski are you OK with this PR?

**Comment by piotr-s-brainhub** [comment]
@redzumi 

Can I ask you to fill in this form https://docs.google.com/forms/d/1tfRs1Io49tJBwz-06aDnz8hrJ3WySJTi45y4k6hHGaQ ?

**Pull request merged**